core/server/worker/pom.xml

```diff
@@ -102,6 +102,12 @@
       <artifactId>alluxio-integration-fuse</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <!-- alluxio.master.metastore.rocks.RocksStore is needed -->
```

Comment:
hmmm, if it's too difficult to move the rocksStore to common, I'm ok with the change here, because we will have a brand new branch and impl of dora in the near future

Reply:
When I added this dependency, I tried to move this class to common.
But it seemed a bit complicated.

I will try to do that again.

Reply:
Done.